accelerator/roles/intel/vars/main.yml

```diff
@@ -22,14 +22,14 @@ intel_gaudi_kernel_module_to_load:
 
 # TODO: move to a central config file
 intel_habana_packages:
-  - habanalabs-dkms
-  - habanalabs-firmware
-  - habanalabs-firmware-tools
-  - habanalabs-graph
-  - habanalabs-qual
-  - habanalabs-rdma-core
-  - habanalabs-thunk
-  - habanatools
+  - "habanalabs-dkms={{ intelgaudi_version }}"
```

Comment:
This should include `habanalabs-firmware-odm`. This enables `hl-fw-loader` in case the admin wants to upgrade the Intel Gaudi FW. I know that Omnia don't do FW upgrades but this should be available to the sys_admins so they can do it.